src/Servers/HttpSys/src/RequestProcessing/Request.cs

```diff
@@ -351,6 +354,34 @@ private AspNetCore.HttpSys.Internal.SocketAddress LocalEndPoint
         }
     }
 
+    public ReadOnlyCollection<long> RequestTimestamps
+    {
+        get
+        {
+            if (_requestTimestamps == null)
+            {
+                /*
+                    Below is the definition of the timing info structure we are accessing the memory for.
+                    We can skip the first ULONG since it's always set to HttpRequestTimingTypeMax which is the size of the array.
+
+                    TODO: I would expect 244 bytes (4 bytes + 8 bytes * 30 [current size of HttpRequestTimingTypeMax]) but it's 248.
```

Comment:
Anyone have thoughts on why this is?

Reply:
Because ulong in C# is 64 bits?
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/integral-numeric-types

Reply:
But this struct is defined in c. 

So I think it should be 32 bit - https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-dtyp/32862b84-f6e6-40f9-85ca-c4faf985b822

Otherwise why define the count as ULONG and the timestamps as ULONGLONG?

Reply:
Figured this out. It has to do with the compiler enforcing alignment of the data within the struct, so it adds 4 bytes of passing after the ULONG.